.github/workflows/testing-pipline.yml

```diff
@@ -0,0 +1,132 @@
+name: Integration-test
+
+on:
+  pull_request:
+
```

Comment:
Add a `on.pull_request.paths` and/or  `on.pull_request.paths-ignore` key to limit when this pipeline needs to run.